database-core/database-task.py

```diff
@@ -0,0 +1,170 @@
+from typing import List, Type
+from sqlalchemy import create_engine, Column, String, Integer, update, delete
+from sqlalchemy.orm import registry
+from sqlalchemy.orm import sessionmaker
+import argparse
+import sys
+
+DATABASE_NAME = 'films_db.db'
+TABLE_NAME = 'films'
+
+mapper_registry = registry()
+Base = mapper_registry.generate_base()
+engine = create_engine(f'sqlite+pysqlite:///{DATABASE_NAME}', echo=True)
+Session = sessionmaker(bind=engine)
+
+
+class Film (Base):
+    __tablename__ = TABLE_NAME
+
+    id = Column('id', Integer, primary_key=True, nullable=False)
```

Comment:
I would use autoincrement here. This will make it much easier to work with a database with a large number of records.